payments-ui-core/src/main/java/com/stripe/android/ui/core/forms/resources/LpmRepository.kt

```diff
@@ -61,10 +63,20 @@ class LpmRepository @Inject constructor(
     }
 
     @VisibleForTesting
-    fun initialize(inputStream: InputStream?) {
+    fun initialize(
```

Comment:
You will have to remind me to do this when we read the specs from the server as well.

Reply:
I moved the object to the constructor instead, let me know if that changes anything